### ISSUE TITLE
"Empty" RichTextEdit

### ISSUE THREAD

**Issue by WolfgangKluge** [Megabit/Blazorise#4114]
**Describe the bug**
After setting HTML of RichTextEdit, the edit area still remains empty - most of the times.
Blazorise Version is 1.0.4 and 1.0.6 (still in 1.1.3.1)

**To Reproduce**
I tracked it down to this simple example
```razor
@using Blazorise.RichTextEdit

<RichTextEdit @ref="editor"
    Theme="RichTextEditTheme.Snow">
</RichTextEdit>

@code {
    private RichTextEdit? editor;

    private string html = "Hello world.";
    private bool isSet;

    protected override async Task OnAfterRenderAsync(bool firstRender)
    {
        if (!isSet && editor != null)
        {
            isSet = true;

            await editor.SetHtmlAsync(html);
            StateHasChanged();
        }
        await base.OnAfterRenderAsync(firstRender);
    }
}
```

The only thing this code does is to call `SetHtmlAsync` on startup.
After opening the page, "Hello world" should be visible.

**Expected behavior**
The text is changed after `SetHtmlAsync`

**Additional context**
In my tests it worked far way more often (if not always), if I
a) remove `StateHasChanged();` in `OnAfterRenderAsync`, or
b) remove `Theme="RichTextEditTheme.Snow"`

both of which are inexplainable to me (especially `b)` since `Snow` is the default value)

**Comment by WolfgangKluge** [comment]
I've looked into detail somewhat further and stumbled upon `RichTextEdit.OnFirstAfterRenderAsync`.

As I can see, the order of execution between `OnFirstAfterRenderAsync` and `ExecuteAfterRender`-Tasks are not guaranteed. Thus, the `JSModule.SetHtmlAsync` is called before `JSModule.Initialize` is finished (and is even executed in parallel)

**Comment by WolfgangKluge** [comment]
Cannot find the exact position with the bug, yet, but if I wait inside `JSRichTextEditModule.SetHtmlAsync` for `JSRichTextEditModule.Initialize` to finish, then it works as excepted..

**Comment by WolfgangKluge** [comment]
Sometimes there are additional javascript errors. Maybe (don't know) that's an additional problem with the `ExecuteAfterRender`-queue.

To prevent these errors, you have to change all the references to `editorRef.quill` in [richtextedit.js](https://github.com/Megabit/Blazorise/blob/master/Source/Extensions/Blazorise.RichTextEdit/wwwroot/richtextedit.js#L108) and check, if `editorRef` is not null (that happens, when the [id is not found](https://github.com/Megabit/Blazorise/blob/master/Source/Extensions/Blazorise.RichTextEdit/wwwroot/richtextedit.js#L6)).

**Comment by stsrki** [comment]
So it seems we will need to postpone the execution of those calls or add them to the queue after the initialization. Thanks for looking into it.
